packages/functional-tests/tests/react-conversion/resetPassword.spec.ts

```diff
@@ -20,31 +16,29 @@ test.describe('reset password', () => {
     test.skip(config.showReactApp.resetPasswordRoutes !== true);
   });
 
-  test('can reset password', async ({ page, target, credentials, context }) => {
+  test('can reset password', async ({
+    page,
+    target,
+    credentials,
+    context,
+    pages: { login, resetPasswordReact },
+  }) => {
     await page.goto(getReactFeatureFlagUrl(target, '/reset_password'));
```

Comment:
nit: You might be able to add `goto` function in Password Reset React that automatically appends required query params needed.